cai_causal_graph/causal_graph.py

```diff
@@ -534,6 +542,75 @@ def is_empty(self) -> bool:
         """Return True if there are no nodes and edges. False otherwise."""
         return len(self.nodes) == 0 and len(self.edges) == 0
 
+    def _check_node(self, identifier: NodeLike) -> str:
+        identifier = Node.identifier_from(identifier)
```

Comment:
Docstring

Reply:
Also `_check_node` is an ambiguous name, I would rename it something more specific to what you're doing

Reply:
Maybe `_check_node_exists` or `_verify_node`? I agree with Max on the name ambiguity